doc/conf_ref.rst

```diff
@@ -143,13 +143,24 @@ or :ref:`mirrorlist <mirrorlist-label>` option definition.
 ``pluginconfpath``
     :ref:`list <list-label>`
 
-    List of directories that are searched for plugin configuration to load. All configuration files found in these directories, that are named same as a plugin, are parsed. The default contains ``/etc/dnf/plugins`` path.
+    List of directories that are searched for plugin configurations to load. All
+    configuration files found in these directories, that are named same as a
+    plugin, are parsed. The default path is ``/etc/dnf/plugins``.
 
 ``pluginpath``
     :ref:`list <list-label>`
 
     List of directories that are searched for plugins to load. Plugins found in *any of the directories* in this configuration option are used. The default contains a Python version-specific path.
 
+``reposdir``
+    :ref:`list <list-label>`
+
+    DNF searches in the paths specified by reposdir for repository files. If the \-\
```

Comment:
DNF searches for repository configuration files in the paths specified by `reposdir`.